jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandler.java

```diff
@@ -149,10 +146,10 @@ public static Context getCurrentContext()
     /* ------------------------------------------------------------ */
     public static ContextHandler getContextHandler(ServletContext context)
     {
-        if(context instanceof ContextHandler.Context)
+        if (context instanceof ContextHandler.Context)
```

Comment:
file appears to have been reformatted.  I typically don't like that to be mixed in with behaviour changes.... but it's done now....

Reply:
Sorry yes I agree.  I usually do as a separate commit also but missed this.  I think we have both been fixing at the same time,  May have a merge issue.  Let me check your changes.